openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -865,8 +866,10 @@ Here is another example JWT not only proving possession of a private key but als
 
 ```json
 {
+  "typ": "openid4vci-proof+jwt",
   "alg": "ES256",
-  "x5c": [<key certificate + certificate chain for attestation>]
+  "kid": 0,
```

Comment:
```suggestion
  "kid": "0",
```

This header parameter is a string as per Section 4.1.4 of RFC 7515 https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.4

Reply:
I do have a seperate question as to why we included this parameter in the header of this example, what purpose is it serving?

Reply:
I was merely fixing an incorrect example in the spec and "kid" is one of the options